taskFive.java

```diff
@@ -0,0 +1,13 @@
+// napisat' class kotoruy vuvodit v consol' bol'shee iz dvyh zadanuh chisel
+
+class taskFive {
+    public static void main(String[] args) {
+        int valueOne = 5;
+        int valueTwo = 10;
+        if (valueOne > valueTwo) {
+            System.out.println("largest number: " + valueOne);
+        } else if (valueOne < valueTwo) {
+            System.out.println("largest number: " + valueTwo);
+        }
```

Comment:
To be more correct you also need a case when numbers are equal

Reply:
fixed